wpilibc/src/main/native/include/frc/DriverStation.h

```diff
@@ -266,9 +266,33 @@ class DriverStation : public ErrorBase {
   bool IsBrownedOut() const;
 
   std::string GetGameSpecificMessage() const;
+  /**
+   * Returns the name of the competition event from the FMS.
+   *
+   * @return An std::string containing the event name
+   */
   std::string GetEventName() const;
+
+  /**
+   * Return the type of match being played from the FMS.
+   *
+   * @return The match type enum (kNone, kPractice, kQualification,
+   * kElimination)
```

Comment:
You'll want to indent this for readability:
```suggestion
   *         kElimination)
```

Reply:
I could go either way here.  Usually for me this is fine because we cannot enforce it will the tools right now.